src/app/util/conversion_to_surface.cpp

```diff
@@ -250,6 +250,9 @@ void convert_image_to_surface(const doc::Image* image,
   os::SurfaceLock lockDst(surface);
   os::SurfaceFormatData fd;
   surface->getFormat(&fd);
+  // Needed because Skia surfaces work with premultiplied alpha and
+  // here we need to copy unpremultiplied alpha RGB values to the Skia surface.
+  ((os::SkiaSurface*)surface)->bitmap().setAlphaType(kUnpremul_SkAlphaType);
```

Comment:
We need to enclose this code between `LAF_SKIA` so the compilation doesn't fail when `LAF_BACKEND=none`:
```suggestion

#if LAF_SKIA
  // Needed because Skia surfaces work with premultiplied alpha and
  // here we need to copy unpremultiplied alpha RGB values to the Skia surface.
  ((os::SkiaSurface*)surface)->bitmap().setAlphaType(kUnpremul_SkAlphaType);
#endif
```